spritz/app/src/main/res/layout/main.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why do you even have `ConstraintLayout` in your dependencies if then you end up using a `RelativeLayout` here? I'd convert to `ConstraintLayout` and simplify this (you won't need the `LinearLayout` anymore)

Reply:
Removed dependency in bf8af393e8ffd08e0294948383ba373bde63e63e.

Reply:
I would have done the other way around but hey same thing 💃